tigris/example_test.go

```diff
@@ -63,7 +63,7 @@ func ExampleOpenDatabase() {
 	// Connects to the Tigris server. Creates or opens database "db1".
 	// Creates or migrate &Coll1{}. Returns a "db" object, which provides
 	// access to the collections of the database, Coll1 in this example.
-	db, err := OpenDatabase(ctx, &Config{Project: "db1"}, "db1", &Coll1{})
+	db, err := OpenDatabase(ctx, &Config{}, "db1", &Coll1{})
```

Comment:
We need to move db1 inside tigris.Config?

Reply:
yes this was left. thanks!